src/library_manager/Kconfig

```
@@ -7,6 +7,7 @@ menu "Library Manager"
 config LIBRARY_MANAGER
 	bool "Library Manager Support"
 	default n
+	depends on CONFIG_LLEXT
```

I don't think this is correct. The library manager can also be used for LMDK and IADK modules, which don't need LLEXT

---

It was even more incorrect because it should have been:
```suggestion
	depends on LLEXT
```